documentation/setup/download.mdx

```diff
@@ -89,3 +89,6 @@ wget -O - https://get.hacs.xyz | bash -
 
 - Having issues accessing `https://get.hacs.xyz` in the terminal?
   - Try it with `https://raw.githubusercontent.com/hacs/get/main/get` instead
+
+- Getting `ERROR: 'unzip' is not installed`?
```

Comment:
TBH I do not really want this.
It only makes sense for "core" installations, and IMO if you can not manage this, you should not be using core, and not be using HACS.